### PR TITLE
Naming for the nodes during the import from ONNX

### PR DESCRIPTION
**Pull request by aclex**
Node names are set during the importing the graph from ONNX. The naming based on the optional (yet pretty much always there) [node names](https://github.com/onnx/onnx/blob/31f0fd6103e231664d1cfd3b25264f672f5de960/onnx/onnx.proto#L202) in the original graph or on the total number of nodes in the graph at the moment (to get unique name for each next node created).

The overall scheme is like this:

**`2[.3](1)`**

where
1. ONNX operation type (like `Gemm` or `Add`).
2. Extracted or generated node name (see above).
3. Micro-operation created during the formula implementation for certain complex ONNX operations (like in `BatchNormalization`, `Gemm` etc.), if applicable.

Example: `Relu_1.max(Relu)`.

### CODE REVIEW

File: src/importer/onnx/onnx_importer.cpp [hunk 1]
```diff
@@ -788,3 +788,11 @@ shape_t onnx_importer::broadcast_shape(const shape_t &v_shape, const shape_t &in
 
     return result;
 }
+
+std::string onnx_importer::generate_name(const onnx::NodeProto &node) const
+{
+	if (node.has_name())
```
Comment (sunnycase) [inline src/importer/onnx/onnx_importer.cpp:794]: Use spaces instead of tabs.

**Review by sunnycase — COMMENTED** [review]
Thank you for your contribution!
There are some ident issue in your sources, you can use clang-format to solve them.

File: src/importer/onnx/onnx_importer.cpp (hunk 1 above)
Comment (aclex) [reply src/importer/onnx/onnx_importer.cpp:794]: Oh, sorry for that. Indeed I pay special attention to fixing things like this, but missed this one. Thank you, will add clang-format to my pipeline to have it fixed automatically.

**Review by sunnycase — APPROVED** [review]

**Comment by aclex** [comment]
@sunnycase many thanks for the approvement!

**Pull request merged**